prep-javascript-functions/main.js

```diff
@@ -0,0 +1,57 @@
+function addTwoNumbers(num1, num2) {
+  return num1 + num2;
+}
+const addTwoNumbersResult = addTwoNumbers(2, 2);
+console.log('Add two numbers exercise: ' + addTwoNumbersResult);
+function convertHourstoMinutes(hours) {
+  return hours * 60;
+}
+const Hours = convertHourstoMinutes(2);
```

Comment:
you want to typically avoid uppercase variable names in javascript, there are typically only used for Classes and Constructor functions

Reply:
I pushed the fixes